public/js/navigator-js/integration/ReactRecipe.js

```diff
@@ -0,0 +1,86 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var ReactRecipe = {
+  _type: 'REACT',
+
+  // Returns a fake proxy element due to the fact that
+  // we do not use these transition functions within
+  // our react components
+
+  getViewInstance: function getViewInstance() {
+    return {
+      navigatorBehaviors: ['IHasStateTransition'],
+      transitionIn: function(cb) { cb() },
+      transitionOut: function(cb) { cb() }
+    }
+  },
+
+  // Save reference to our react element instead of
+  // view instance like backbone recipe does
+
+  initialize: function initialize() {
+    var params = this._viewArguments;
+
+    var props = Object.assign(
+      {
+        ref: function(c) {
```

Comment:
`ref` is a function that is passed the reference to the react component instance when it changes.  Similar to what would be returned from a `new Backbone.View()`